src/viam/sdk/common/proto_type.hpp

```diff
@@ -27,64 +27,82 @@ struct get_helper {
     }
 };
 
-template <>
-struct get_helper<AttributeMap> {
-    template <typename V>
-    static AttributeMap get(V& v) {
-        auto* const result = boost::get<AttributeMap>(&v);
-        return result ? *result : nullptr;
-    }
-
-    template <typename V>
-    static std::shared_ptr<const AttributeMap::element_type> get(const V& v) {
-        auto* const result = boost::get<AttributeMap>(&v);
-        return result ? *result : nullptr;
-    }
-};
-
-}  // namespace prototype_details
+}  // namespace value_type_details
```

Comment:
The purpose of `get_helper` was to specialize for the `AttributeMap` case to deal with the extra layer of pointer with an automatic dereference in the populated case. Since the specialization has been removed, it suggests that `get_helper` is no longer needed. But I have a suspicion it still may need to exist, but with different specializations.